bundles/org.openhab.binding.opengarage/src/main/java/org/openhab/binding/opengarage/internal/api/ControllerVariables.java

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.opengarage.internal.api;
+
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Class for holding the set of parameters used to read the controller variables.
+ *
+ * @author Paul Smedley - Initial Contribution
+ *
+ */
+public class ControllerVariables {
+    private static Logger LOGGER = LoggerFactory.getLogger(ControllerVariables.class);
+
+    public Integer dist;
+    public Integer door;
+    public Integer vehicle;
+    public Integer rcnt;
+    public Integer fwv;
+    public String name;
+    public String mac;
+    public String cid;
+    public Integer rssi;
+
+
+    private ControllerVariables() {
+    }
+
+    public static ControllerVariables parse(String response) {
+        LOGGER.debug("Parsing string: \"{}\"", response);
+        /* parse json string */
+        JsonObject jsonObject = new JsonParser().parse(response).getAsJsonObject();
+        ControllerVariables info = new ControllerVariables();
+        info.dist = jsonObject.get("dist").getAsInt();
```

Comment:
The `getAsInt()` returns a primitive `int`. So the field member can all be primitives `int` instead of object `Integer`.

Reply:
Fixed in 7184ad28bb